data/map.txt

```diff
@@ -22490,7 +22490,8 @@ system Rutilicus
 	government Republic
 	attributes "dirt belt"
 	arrival 625
-	habitable 625
+	departure 2000
+	habitable 2000
```

Comment:
Do we want to have this in-game for Rutilicus? Or is this a testing-artefact that got left behind?